08.ls_object/lib/command.rb

```diff
@@ -13,15 +13,15 @@ def initialize(dotmatch: false, reverse: false, long_format: false)
     end
 
     def exec
-      sorted_paths = sort_paths
+      collected_paths = collect_paths
```

Comment:
```suggestion
      paths = collect_paths
```

「素のpaths」ということで何もつけなくてもいいかもしれません